src/demo-app/data-table/data-table-demo.scss

```diff
@@ -0,0 +1,86 @@
+/* Structure */
+.demo-table-container {
+  display: flex;
+  flex-direction: column;
+  max-height: 800px;
+
+  // Table fills in the remaining area with a scroll
+  .cdk-table {
+    flex: 1 1 auto;
+    overflow: auto;
+  }
+}
+
+/* Material Styles */
```

Comment:
Context on why Material styles are in the demo CSS?  I assume the component doesn't come with styles, so this is just to make it look Material-y? The comment makes it sound like this comes with the component.